CHANGELOG.md

```diff
@@ -3,8 +3,12 @@ All notable changes to this project will be documented in this file.
 
 The format is based on [Keep a Changelog](http://keepachangelog.com/en/1.0.0/) and this project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.html).
 
-## [4.1.0] - 2019-02-27
+## [4.1.1] - 2019-04-30
+### Changed
+- Default supported Hive version is now 2.3.4 (was 2.3.3).
+-- Version 2.3.3 has a vulnerability: https://nvd.nist.gov/vuln/detail/CVE-2018-1314
```

Comment:
I don't think `--` does an intent on GH markdown. I'll fix this here. Also, we don't put dates in here until we know for sure we're going to do a release (unless you're planning on doing one today?)